tests/ui/multiple_arbitrary.stderr

```diff
@@ -1,7 +1,5 @@
 error: Multiple conflicting #[arbitrary] attributes found on field `x`
  --> tests/ui/multiple_arbitrary.rs:5:5
   |
-5 | /     #[arbitrary(value = 2)]
-6 | |     #[arbitrary(value = 3)]
-7 | |     x: i32,
-  | |__________^
+5 |     #[arbitrary(value = 2)]
+  |     ^
```

Comment:
Does this mean that the tests won't pass on both nightly and stable at the same time right now?

Reply:
That's correct. I got one tests broken on CI, because I was running `nightly` locally, but CI was running `stable`.

OK, i'll change it to doc tests then.